frontend/awx/access/credentials/CredentialForm.tsx

```diff
@@ -295,38 +404,18 @@ function CredentialSubForm({ credentialType }: { credentialType: CredentialType
   const requiredFields = credentialType?.inputs?.required || [];
 
   const hasFields = stringFields.length > 0 || choiceFields.length > 0 || booleanFields.length > 0;
-
   return hasFields ? (
     <PageFormSection title={t('Type Details')}>
       {stringFields.length > 0 &&
-        stringFields.map((field) => {
-          if (field?.multiline) {
-            return (
-              <CredentialMultilineInput
-                kind={credentialType.kind}
-                key={field.id}
-                field={field}
-                requiredFields={requiredFields}
-              />
-            );
-          } else if (credentialType.kind === 'ssh' && field.id === 'become_method') {
```

Comment:
This should not be removed...it is an edge case and we should keep this case.

Reply:
Agreed, I'll put that back in. Does this case also have the potential to be an encrypted field?

Reply:
@daphnemaeve it will have to manual test that one.